### PR TITLE
Improved Docs 

### PR DESCRIPTION
**Pull request by aman1750**

### CODE REVIEW

**Comment by smirolo** [comment]
Thanks!

**Comment by toloveru** [comment]
This is a single character change, encouraged in https://www.youtube.com/watch?v=v9902nonYas and now one of many spam pull requests. Could you add the spam label to this PR so that DigitalOcean doesn't count it towards their Hacktoberfest event?

**Comment by smirolo** [comment]
Reverted the change. The video link is marked as private so I am not sure what is in it and/or why you want to consider it spam. labelled it as such anyway. We have a welcome policy to accept all pull requests that do not introduce functional errors or security issues from first time contributors.

**Comment by toloveru** [comment]
The video used to be a popular YouTuber suggesting people to make low quality contributions to "game the system" on DigitalOcean's Hacktoberfest - the backlash must be why it's private now. DigitalOcean acknowledged the issue on https://hacktoberfest.digitalocean.com/hacktoberfest-update and it's been covered in https://www.theregister.com/2020/10/01/digitalocean_hacktoberfest_pull_request_spam as well.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Tested with
 - **Python:** 3.6, **Django:** 3.0 (latest), **Django Rest Framework:** 3.11
 
 
-Development
+Development :
```
Comment (robbyoconnor) [inline README.md:17]: What in the literal fuck?

**Comment by robbyoconnor** [comment]
I'd just like to thank you morons for ruining Hacktoberfest.

File: README.md (hunk 1 above)
Comment (KapoorDev) [reply README.md:17]: Lol a space and semi colen 😅

**Comment by smirolo** [comment]
We have an inclusive policy to accept all harmless changes from first time contributors even if it means we will revert them later on. It is OK to have heated discussion about technical or policy issues. What we will not accept is people insulting each other personally.

**Pull request merged**